src/main/kotlin/com/team4099/robot2023/config/constants/VisionConstants.kt

```diff
@@ -23,4 +23,6 @@ object VisionConstants {
         Rotation3d(180.degrees, 0.degrees, 0.degrees)
       )
     )
+
+  val CAMERA_NAMES = listOf("Barry", "Wells", "Cisco", "Caitlin")
```

Comment:
-_-

Reply:
also these names don't match the northstar instance names above?

Reply:
oops i was trolling and forgot to change